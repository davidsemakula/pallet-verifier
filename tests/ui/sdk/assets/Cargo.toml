[package]
name = "pallet-assets"
version = "29.1.0"
#authors.workspace = true
#edition.workspace = true
edition = "2021"
license = "Apache-2.0"
#homepage.workspace = true
#repository.workspace = true
description = "FRAME asset management pallet"
readme = "README.md"

#[lints]
#workspace = true

[package.metadata.docs.rs]
targets = ["x86_64-unknown-linux-gnu"]

[dependencies]
codec = { version = "3.6.12", default-features = false, package = "parity-scale-codec" }
impl-trait-for-tuples = "0.2.2"
log = { version = "0.4.22", default-features = false }
scale-info = { version = "2.11.3", default-features = false, features = ["derive"] }
# Needed for various traits. In our case, `OnFinalize`.
sp-runtime = { version = "39.0.0", default-features = false }
# Needed for type-safe access to storage DB.
frame-support = { version = "38.0.0", default-features = false }
# `system` module provides us with all sorts of useful stuff and macros depend on it being around.
frame-system = { version = "38.0.0", default-features = false }
frame-benchmarking = { version = "38.0.0", optional = true, default-features = false }
sp-core = { version = "34.0.0", default-features = false }

[dev-dependencies]
sp-io = { version = "38.0.0", default-features = true }
pallet-balances = { version = "39.0.0", default-features = true }

[features]
default = ["std"]
std = [
	"codec/std",
	"frame-benchmarking?/std",
	"frame-support/std",
	"frame-system/std",
	"log/std",
	"pallet-balances/std",
	"scale-info/std",
	"sp-core/std",
	"sp-io/std",
	"sp-runtime/std",
]
runtime-benchmarks = [
	"frame-benchmarking/runtime-benchmarks",
	"frame-support/runtime-benchmarks",
	"frame-system/runtime-benchmarks",
	"pallet-balances/runtime-benchmarks",
	"sp-runtime/runtime-benchmarks",
]
try-runtime = [
	"frame-support/try-runtime",
	"frame-system/try-runtime",
	"pallet-balances/try-runtime",
	"sp-runtime/try-runtime",
]
