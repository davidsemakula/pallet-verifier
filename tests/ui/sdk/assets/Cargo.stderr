warning: possible attempt to add with overflow
  --> src/impl_fungibles.rs:255:3
   |
LL |         Self::calc_metadata_deposit(&name, &symbol)
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
note: related location
  --> src/functions.rs:1019:21
   |
LL |             .saturating_mul(((name.len() + symbol.len()) as u32).into())
   |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^

